R/df_to_exb.r

```diff
@@ -0,0 +1,340 @@
+#' Dataframe to .exb
+#'
+#' Convert a dataframe to Exmaralda .exb
+#'
+#' @author Valeria Buntiakova <valleriabun@gmail.com>
+#'
+#' @param df an R dataframe object that contains columns named 'tier', 'tier_name', 'content', 'time_start', 'time_end' and 'id'
+#' @param name transcription name
```

Comment:
I need a more elaborate description here...

Reply:
It is what it says: the name one wishes to give the transcription they are writing. It's part of meta information, so we can take it with all the other meta information in a vector. It's not obligatory, and I can either move it or get rid of it.